src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -3277,6 +3277,164 @@ public static String substringBetween(final String str, final String open, final
         return list.toArray(new String[list.size()]);
     }
 
+    /**
+     * <p>Split a String into an array, using an array of fixed string lengths.</p>
+     *
+     * <p>If not null String input, the returned array size is same as the input lengths array.</p>
+     *
+     * <p>A null input String returns {@code null}.
+     * A {@code null} or empty input lengths array returns an empty array.
+     * A {@code 0} in the input lengths array results in en empty string.</p>
+     *
+     * <p>Extra characters are ignored (ie String length greater than sum of split lengths).
+     * All empty substrings other than zero length requested, are returned {@code null}.</p>
+     *
+     * <pre>
+     * StringUtils.splitByLength(null, *)      = null
+     * StringUtils.splitByLength("abc")        = []
+     * StringUtils.splitByLength("abc", null)  = []
+     * StringUtils.splitByLength("abc", [])    = []
+     * StringUtils.splitByLength("", 2, 4, 1)  = [null, null, null]
```

Comment:
I followed the same logic as in other `StringUtils.split*(...)` methods : return `null` if the input string is `null`.
So `StringUtils.splitByLength(null, *)` return `null`